Extend the main page with new links and details (#5506)
ppy/osu-wiki: wiki/Main_Page/en.md

<!-- Do not add any empty lines inside this div. -->

<div class="wiki-main-page__blurb">
Welcome to the osu! wiki, the open-source knowledge base containing a wide range of information related to osu!, a free-to-win rhythm game.
</div>

<div class="wiki-main-page__panels">
<div class="wiki-main-page-panel wiki-main-page-panel--full">

# Getting started

> Important articles and guides which will help you dive into osu!. In case anything goes wrong or appears uncertain, make sure to visit the [Help forum](https://osu.ppy.sh/forum/5).

Reviewer: ```suggestion
Important articles and guides that will help you dive into osu!. In case anything goes wrong or appears uncertain, make sure to visit the [Help forum](https://osu.ppy.sh/forum/5).
```

[Rules](/wiki/Rules) • [Installation](/wiki/Installation) • [Registration](/wiki/Registration) • [Help Centre](/wiki/Help_Centre)

[Chat console](/wiki/Chat_Console) • [BanchoBot](/wiki/BanchoBot) • [FAQ](/wiki/FAQ) • [Guides](/wiki/Guides) • [Glossary](/wiki/Glossary)

</div>
<div class="wiki-main-page-panel">

# Game client

> Being the guide to the world of rhythm, beats, and flashes, osu! has a lot to present. Get to know it better, from miscellaneous settings and customization possibilities to technical details and file formats it uses and understands.

Reviewer 2: ```suggestion
Being the guide to the world of rhythm, beats, and flashes, osu! has a lot to present. Get to know it better, from various settings, through customization capabilities, to technical details and file formats the game uses.
```

> Being the guide to the world of rhythm, beats, and flashes, osu! has a lot to present. Get to know it better, from various settings, through customization capabilities, to technical details and file formats the game uses.

Reviewer: That second sentence is a bit confusing. I'm not even sure what you're trying to say here - if you're just doing a general presentation, try this:
`Get to know the game better from its various settings and customisation capabilities to technical details and file formats it uses.`

# Gameplay

> Anything about clicking circles, striking drums, catching fruits, or touching the synthesizer: key elements, concepts, mechanics, and competitive play.

Reviewer 2: ```suggestion
Everything about clicking circles, striking drums, catching fruits, and playing the synthesizer: key elements, concepts, mechanics, and competitive play.
```

[Game modes](/wiki/Game_mode): [osu!](/wiki/Game_mode/osu!) • [osu!taiko](/wiki/Game_mode/osu!taiko) • [osu!catch](/wiki/Game_mode/osu!catch) • [osu!mania](/wiki/Game_mode/osu!mania)

[Beatmap](/wiki/Beatmap) • [Difficulty](/wiki/Beatmap/Difficulty) • [Hit object](/wiki/Hit_object) • [Mods](/wiki/Game_modifier) • [Score](/wiki/Score) • [Replay](/wiki/Replay) • [Medals](/wiki/Medals) • [Multi](/wiki/Multi)

</div>
<div class="wiki-main-page-panel">

# [Beatmap editor](/wiki/Beatmap_Editor)

> All osu! beatmaps are created by its vibrant and dynamic community, blooming with ideas. Always wanted to share your favourite music with others, but never knew how? Familiarize yourself with the built-in editor and the opportunities it has to offer.

Reviewer: `
```suggestion
All osu! beatmaps are created by its vibrant and dynamic community blooming with ideas. Always wanted to share your favourite music with others, but never knew how? Familiarize yourself with the built-in editor and the opportunities it has to offer.
```

The first sentence is awkward to me, although it kind of works. I'd still suggest replacing it with something along the lines of `All osu! beatmaps are created by the blooming imagination of its dynamic community.`

Sections: [Compose](/wiki/Beatmap_Editor/Compose) • [Design](/wiki/Beatmap_Editor/Design) • [Timing](/wiki/Beatmap_Editor/Timing) • [Song setup](/wiki/Beatmap_Editor/Song_Setup)

Components: [AiMod](/wiki/Beatmap_Editor/AiMod) • [Beat snap divisor](/wiki/Beatmap_Editor/Beat_Snap_Divisor) • [Distance snap](/wiki/Beatmap_Editor/Distance_Snap) • [Menu](/wiki/Beatmap_Editor/Menu) • [SB load](/wiki/Beatmap_Editor/SB_Load) • [Timelines](/wiki/Beatmap_Editor/Timelines)

Activities: [Beatmapping](/wiki/Beatmapping) • [Mapping techniques](/wiki/Mapping_Techniques) • [Storyboarding](/wiki/Storyboarding) • [Skinning](/wiki/Skinning)

</div>
<div class="wiki-main-page-panel">

# Beatmap submission and ranking

> Ranking is the process of obtaining a scoreboard for a beatmap, which makes it more popular and well-recognized. Discover what it takes to walk this path, and how to increase your chance to succeed: beatmap reviews, quality control, and procedures that help streamline ranking.

Reviewer: Check "scoreboard" for consistency; iirc it's called "leaderboard" everywhere

> Ranking is the process of obtaining a scoreboard for a beatmap, which makes it more popular and well-recognized. Discover what it takes to walk this path, and how to increase your chance to succeed: beatmap reviews, quality control, and procedures that help streamline ranking.

Reviewer: The second sentence is also very gauche: it groups together two things that should be separate (the modding part and the quality control part). Try something like this:

`Discover what it takes to walk this path and how to increase your chances to succeed through beatmap reviews, as well as the procedures for quality control and the ranking process.`

Though tbh even that one feels off, I'll mull a better one over

Author: better than mine, so I'll take it

[Submission](/wiki/Submission) • [Modding](/wiki/Modding) • [Ranking procedure](/wiki/Beatmap_ranking_procedure) • [Mappers' Guild](/wiki/Mappers_Guild) • [Project Loved](/wiki/Project_Loved)

[Ranking criteria](/wiki/Ranking_Criteria): [osu!](/wiki/Ranking_Criteria/osu!) • [osu!taiko](/wiki/Ranking_Criteria/osu!taiko) • [osu!catch](/wiki/Ranking_Criteria/osu!catch) • [osu!mania](/wiki/Ranking_Criteria/osu!mania)

</div>
<div class="wiki-main-page-panel">

# Rules and legalese

Read about procedures and regulations that keep osu! and its community safe, from music licensing and media guidelines to chat and forum rules.

[Legal pages](/wiki/Legal): [Copyright (DMCA)](/wiki/Legal/Copyright) • [Music licensing](/wiki/Legal/Music_Licensing) • [Privacy](/wiki/Legal/Privacy) • [Terms of Service](/wiki/Legal/Terms)

Media regulations: [Content usage guidelines](/wiki/Rules/Content_Usage_Guidelines) • [Explicit content](/wiki/Rules/Explicit_Content) • [Song content rules](/wiki/Rules/Song_Content_Rules) • [Visual content considerations](/wiki/Rules/Visual_Content_Considerations)

> [Rules](/wiki/Rules) • [Contributor Code of Conduct](/wiki/Contributor_Code_of_Conduct) • [Code of Conduct for modding and mapping](wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping)

Reviewer 2: i think the order for those 3 lines should be `312` instead?

Author: how about 3-2-1, most to least needed usually?

> [Legal pages](/wiki/Legal): [Copyright (DMCA)](/wiki/Legal/Copyright) • [Music licensing](/wiki/Legal/Music_Licensing) • [Privacy](/wiki/Legal/Privacy) • [Terms of Service](/wiki/Legal/Terms)

Reviewer: This is a bit picky but I'd reorder these to be like Terms of Service, Music Licensing, Copyright, Privacy

Reviewer 3: Should probably match the footer if anything:

![20210729 152918 (Safari)](https://user-images.githubusercontent.com/191335/127442750-fc39e72b-6131-4f35-8613-7d16937b2dc4.png)

</div>
<div class="wiki-main-page-panel">

# Help

> Answers to frequent questions and solutions to common issues live here. Learn how to deal with network issues, or combat bad behaviour online.

Reviewer 2: ```suggestion
Answers to frequent questions and solutions to common issues live here. Learn how to deal with network issues, or combat bad behaviour in-game.
```

Author: this is not limited to the game itself, but also other places like forums (besides, the abuse handling guide extends [even beyond that](https://osu.ppy.sh/wiki/en/Reporting_Bad_Behaviour/Abuse))

Reviewer 2: how about
```suggestion
Answers to frequent questions and solutions to common issues live here. Learn how to deal with network issues, or combat bad behaviour in the community.
```

Author: yeah that works

> Answers to frequent questions and solutions to common issues live here. Learn how to deal with network issues, or combat bad behaviour in the community.

Reviewer: I'd remove the second half or make it far more generic. You've picked two very specific examples of the many articles that are under the help centre.

> [Help Centre](/wiki/Help_Centre) • [Performance troubleshooting](/wiki/Performance_Troubleshooting) • [Account support team](wiki/People/The_Team/Account_support_team) • [Help forum](https://osu.ppy.sh/forum/5)

Reviewer 2: Help Centre appears twice on this page, dunno if intentional 
```suggestion
[Help centre](/wiki/Help_Centre) • [Performance troubleshooting](/wiki/Performance_Troubleshooting) • [Account support team](wiki/People/The_Team/Account_support_team) • [Help forum](https://osu.ppy.sh/forum/5)
```

Author: yes, I considered it important enough to be mentioned both in the wide `<div>` and here

> Profile: [Silence](/wiki/Silence) • [Account restrictions](/wiki/Help_Centre/Account_Restrictions)

Reviewer 2: ```suggestion
Profile restrictions: [Silence](/wiki/Silence) • [Account restrictions](/wiki/Help_Centre/Account_Restrictions)
```

Do your part: [How and what to help with](/wiki/How_you_can_help!) • [Report bad behaviour](/wiki/Reporting_Bad_Behaviour) • [Report abuse](/wiki/Reporting_Bad_Behaviour/Abuse) • [Report cheating](/wiki/Reporting_Bad_Behaviour/Handling_Foul_Play)

</div>
<div class="wiki-main-page-panel">

# Community and projects

> Social interaction is one of main things that fuel osu! players around the world. Explore the activities and contests, get in touch with others, or organize the next big event!

Reviewer 2: ```suggestion
Social interaction is one of main things that fuel osu! players all around the world. Explore the activities and contests, get in touch with others, or organize the next big event!
```

> Social interaction is one of main things that fuel osu! players all around the world. Explore the activities and contests, get in touch with others, or organize the next big event!

Reviewer: ```suggestion
Social interaction is one of main things that fuel osu! players all around the world. Explore the activities and contests, get in touch with others, or help organize the next big event!
```

> [Tournaments](/wiki/Tournaments) • [Featured Artists](/wiki/Featured_Artists) • [Beatmap Spotlights](/wiki/Beatmap_Spotlights) • [Contests](/wiki/Contests) • [Community mentorship program](wiki/Community_Mentorship_Program)

Reviewer: It feels a little off for Featured Artists to be in this section. It's not something that users can directly get involved with - it's more of an official project. You might want to move it to the project section, or give it its own section directly.

Author: whoops, yes.

[Projects](/wiki/Projects): [osu!academy](/wiki/osu!academy) • [osu!mapping](/wiki/osu!mapping) • [osu!talk](/wiki/osu!talk)

</div>
<div class="wiki-main-page-panel">

# People

> osu! wouldn't have been possible without many users helping with development, maintenance and community management: their efforts, time, and dedication make the game live and prosper.

Reviewer: ```suggestion
osu! wouldn't have been possible without many users helping with development, maintenance, and community management: their efforts, time, and dedication make the game live and prosper.
```
oxford comma plzzzzzz

# For developers

Interact with osu! via other applications and media, or help it grow further.

[API](/wiki/osu!api) • [Bot account](/wiki/Bot_account) • [Brand identity guidelines](/wiki/Brand_identity_guidelines) • [osu!dev Discord server](/wiki/osu!dev_Discord_server)

</div>
<div class="wiki-main-page-panel">

# Wiki and meta pages

> Just as almost everything out here, the osu! wiki is written and maintained by volunteers like you. Suggest an improvement, report an issue, or even write your own article at the [osu-wiki GitHub repository](https://github.com/ppy/osu-wiki "osu! wiki at GitHub").

Reviewer 2: ```suggestion
Just as almost everything out here, the osu! wiki is written and maintained by volunteers. Suggest an improvement, report an issue, or even write your own article at the [osu-wiki GitHub repository](https://github.com/ppy/osu-wiki "osu! wiki at GitHub").
```
